### PR TITLE
Versioned is gone

### PR DESCRIPTION
**Pull request by slavapestov**

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

File: include/swift/AST/Decl.h [hunk 1]
```diff
@@ -2282,8 +2282,8 @@ class ValueDecl : public Decl {
   /// \sa getFormalAccess
   /// \sa isAccessibleFrom
   AccessScope
-  getFormalAccessScope(const DeclContext *useDC = nullptr,
-                       bool respectVersionedAttr = false) const;
+  getFormalAccessScope(const DeclContext *useDC=nullptr,
+                       bool isUsageFromInline=false) const;
```
Comment (jrose-apple) [inline include/swift/AST/Decl.h]: (repeating comment here) This isn't really the right name either. It doesn't matter where this is being used from; it matters why the question's being asked.

Comment (slavapestov) [reply include/swift/AST/Decl.h]: I can’t really think of anything better. Any suggestions?

Comment (jrose-apple) [reply include/swift/AST/Decl.h]: Maybe just go with a blunt `treatUsableFromInlineAsPublic`.

**Comment by slavapestov** [comment]
@jrose-apple Please take a look.

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Pull request merged**